core/src/main/resources/AppConstants.properties

```diff
@@ -372,7 +372,7 @@ etag.ehcache.diskExpiryThreadIntervalSeconds=600
 
 # Below listed 4 suppression keys should be used with an adapter name to avoid losing any useful configuration warnings.
 # Example: warnings.suppress.transaction.{NameOfTheAdapter}=true
-warnings.suppress.sqlInjections=false
+warnings.suppress.sqlInjections=true
```

Comment:
niet in de appconstants gek, dan is het voor alle ibissen overal altijd
In de deploymentspecifics van iaf-test :)